include/graphtyper/graph/alu_sequences.hpp

```diff
@@ -5,93 +5,263 @@
 
 namespace gyper
 {
-
 std::vector<std::string> static const ALU_SEQUENCES = {
-{"GGCCGGGCGCGGTGGCTCACGCCTGTAATCCCAGCACTTTGGGAGGCCGAGGCGGGAGGATTGCTTGAGCCCAGGAGTTCGAGACCAGCCTGGGCAACATAGCGAGACCCCGTCTCTACAAAAAATACAAAAATTAGCCGGGCGTGGTGGCGCGCGCCTGTAGTCCCAGCTACTCGGGAGGCTGAGGCAGGAGGATCGCTTGAGCCCAGGAGTTCGAGGCTGCAGTGAGCTATGATCGCGCCACTGCACTCCAGCCTGGGCGACAGAGCGAGACCCTGTCTC"},
-{"GGCCGGGCGCGGTGGCTCACGCCTGTAATCCCAGCACTTTGGGAGGCCGAGGCGGGAGGATCACTTGAGCCCAGGAGTTCGAGACCAGCCTGGGCAACATGGTGAAACCCCGTCTCTACAAAAAATACAAAAATTAGCCGGGCGTGGTGGCGCGCGCCTGTAGTCCCAGCTACTCGGGAGGCTGAGGCAGGAGGATCGCTTGAGCCCGGGAGGTCGAGGCTGCAGTGAGCCGTGATCGCGCCACTGCACTCCAGCCTGGGCGACAGAGCGAGACCCTGTCTC"},
```

Comment:
If you want to prevent re-formatting in a particular case like this, you can put 
```
// clang-format off
...
// clang-format on
```
around the block